atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -112,6 +113,14 @@ def run(self):
             self.log.info(str(e))
             self.log.info('Canceling the compose %s', e.compose_id)
             self.odcs_client.cancel_compose(e.compose_id)
```

Comment:
Is this compose cancelled twice?

Reply:
Should the same rules apply as for the other composes, i.e. don't delete composes passed by params?

Reply:
It's not, even if it appears in a following set `cancel_composes` I test state of compose before I delete them via `['wait', 'generating']`